Arrowgene.Ddon.Database/IDatabase.cs

```diff
@@ -476,6 +476,13 @@ bool InsertBBMContentTreasure(
     bool InsertOrUpdateClanBaseCustomization(uint clanId, byte type, uint furnitureId, DbConnection? connectionIn = null);
     bool DeleteClanBaseCustomization(uint clanId, byte type, DbConnection? connectionIn = null);
     List<CDataClanSearchResult> SearchClans(CDataClanSearchParam param, DbConnection? connectionIn = null);
+    bool InsertClanRequest(uint clanId, uint characterId, DbConnection? connectionIn = null);
+    List<CDataClanJoinRequest> GetClanRequestsByCharacter(uint characterId, DbConnection? connectionIn = null);
```

Comment:
Considering there can only be one application, shouldn't these return a single entity instead of a list?